From dbbd070761fa963ecbbdaaaa0e73d7b5ee8d141c Mon Sep 17 00:00:00 2001
From: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Date: Thu, 27 Oct 2022 16:17:51 +0300
Subject: [PATCH 03/49] plat: nxp: add __unused attribute for reset_rtc
 function

When compiling for an EMU target, reset_rtc isn't used and triggers a
compile-time error. Add an __unused attribute to it.

Issue: ALB-9146

Upstream-Status: Pending 

Signed-off-by: Andrei Stefanescu <andrei.stefanescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 plat/nxp/s32/s32g/s32g_bl2_el3.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/nxp/s32/s32g/s32g_bl2_el3.c b/plat/nxp/s32/s32g/s32g_bl2_el3.c
index c5acbee98..bd407c2c3 100644
--- a/plat/nxp/s32/s32g/s32g_bl2_el3.c
+++ b/plat/nxp/s32/s32g/s32g_bl2_el3.c
@@ -73,7 +73,7 @@ static int init_and_setup_pmic(void)
 	return ret;
 }
 
-static void reset_rtc(void)
+static __unused void reset_rtc(void)
 {
 	uint32_t rtc = S32G_RTC_BASE;
 	uint32_t rtcs;
-- 
2.25.1

